### PR TITLE
Fix signal handler issue: signal handler is restored after execution

### PR DESCRIPTION
**Pull request by b-scholz**
Fix issue #427

### CODE REVIEW

**Comment by mmcgr** [comment]
I think we're having problems with the new images that travis just switched to. That will need fixing, independently of this PR, before the tests can be run.

**Comment by mmcgr** [comment]
Actually, the new image may be coincidence. The llvm 4.0 repository doesn't seem to have llvm 4.0 stuff in it just now.

**Comment by b-scholz** [comment]
It seems to work again; note one test was not working well due to different error messages caused by perror(). I have checked in a new version. I hope it will work now.

File: tests/interface/error/driver.cpp [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      prog->run(); 
+
+      // free program 
+      delete prog; 
+
+      // raise signal to check that signal handler is restored
+      raise(SIGINT);
+
+   } else {
+      error("cannot find program load_print");
```
Comment (mmcgr) [inline tests/interface/error/driver.cpp]: This is no longer `load_print`

File: tests/interface/error/driver.cpp [hunk 2]
```diff
@@ -0,0 +1,71 @@
+/*
+ * Souffle - A Datalog Compiler
```
Comment (mmcgr) [inline tests/interface/error/driver.cpp]: Is this license appropriate?

File: tests/interface/error/driver.cpp [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+ * Copyright (c) 2015, Oracle and/or its affiliates. All rights reserved
```
Comment (mmcgr) [inline tests/interface/error/driver.cpp]: Is this license header appropriate? The code shares little with the older interface tests.

**Pull request merged**